lib/core/src/Cardano/Wallet/Api/Link.hs

```diff
@@ -0,0 +1,520 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE NumericUnderscores #-}
+{-# LANGUAGE OverloadedLabels #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE TypeOperators #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# LANGUAGE UndecidableSuperClasses #-}
```

Comment:
This extension also seems to be unused? (I tried compiling without it, and seemed to compile, but perhaps I'm missing something?)

Reply:
Whoops ^.^ ... not the first draft, I went through several ways, so there are leftovers.